CHANGELOG.md

```diff
@@ -3,6 +3,38 @@
 ### Deprecations
 * None
 
+### Enhancements
+* Added the ability to use an existing Realm instance in `RealmProvider` and `createRealmContext`. ([#6714](https://github.com/realm/realm-js/pull/6714))
+```jsx
+// Using RealmProvider
```

Comment:
I thought it'd be nice to include an example in the changelog and I noticed it has been done before but maybe it should go somewhere else.

Reply:
Definitely a good idea. It gives our users a taste of the new feature, and they can dive deeper into the documentation